### PR TITLE
20 levels

### PR DESCRIPTION
**Pull request by mattThousand**

### CODE REVIEW

File: Shikaku/Screens/Extras/ProfileView.swift [hunk 1]
```diff
@@ -28,19 +29,45 @@ final class ProfileView: UIView {
...
         rowStackView.rightAnchor.constraint(equalTo: rightAnchor).isActive = true
         rowStackView.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
         rowStackView.leftAnchor.constraint(equalTo: leftAnchor).isActive = true
     }
 
     func configureSubviews() throws {
         addSubview(rowStackView)
+        (0..<Int.columnCount).forEach { _ in
+            let columnStackView = UIStackView()
+            columnStackView.translatesAutoresizingMaskIntoConstraints = false
```
Comment (ateliercw) [inline Shikaku/Screens/Extras/ProfileView.swift]: #? If these are going inside a `UIStackView` I don't think you need to turn off `translatesAutoresizingMaskIntoConstraints`

File: Shikaku/Screens/Extras/ProfileViewController.swift [hunk 2]
```diff
@@ -10,21 +10,43 @@ import UIKit
...
+    fileprivate var viewModel: ProfileViewModel = ProfileViewModel()
+
+    override var hideNavigationBar: Bool { return false }
+
     override func viewDidLoad() {
         super.viewDidLoad()
         Log.tryAndLog { try configureSubviews() }
         Log.tryAndLog { try configureConstraints() }
+        configureBindings()
+        viewModel.completedLevelCount.onNext(5)
```
Comment (ateliercw) [inline Shikaku/Screens/Extras/ProfileViewController.swift]: #? is this just a placeholder magic number?

File: Shikaku/Screens/Start Menu/ExpandingHamburgerView.swift [hunk 3]
```diff
@@ -111,9 +112,9 @@ class ExpandingHamburgerView: UIView {
         fatalError("init(coder:) has not been implemented")
     }
 
-    func morph() {
+    func morph(with completion: (() -> ())? = nil) {
```
Comment (ateliercw) [inline Shikaku/Screens/Start Menu/ExpandingHamburgerView.swift]: #pp prefer `() -> Void` to `() -> ()`

File: Shikaku/Screens/Extras/ProfileView.swift (hunk 1 above)
Comment (mattThousand) [inline Shikaku/Screens/Extras/ProfileView.swift]: ah, you are right!

File: Shikaku/Screens/Extras/ProfileViewController.swift (hunk 2 above)
Comment (mattThousand) [inline Shikaku/Screens/Extras/ProfileViewController.swift]: yes... adding a  TODO to remove it. Ultimately this number will come from GameCenter/iCloud

File: Shikaku/Screens/Start Menu/ExpandingHamburgerView.swift (hunk 3 above)
Comment (mattThousand) [inline Shikaku/Screens/Start Menu/ExpandingHamburgerView.swift]: fair enough :)

**Pull request merged**